### DISCUSSION TITLE
Is it possible to wrap diagnostic virtual text/make it multiline?

### DISCUSSION THREAD

**Discussion by rudenkornk** [folke/trouble.nvim#595] [Q&A] [answered]
Sometimes virtual text from a diagnostic is too long and is not visible. It would be really nice to wrap it somehow on a screen.
![image](https://github.com/user-attachments/assets/9dec988e-42d2-4374-8435-f514af28f097)

There is already a similar discussion here, but it is not answered: https://github.com/folke/trouble.nvim/discussions/380

**Comment by folke** [comment, marked as answer]
That's not Trouble. The underlines are a natve NEovim feature
